docs/iota-did-method-spec.md

```diff
@@ -0,0 +1,128 @@
+3. `com` references the test network known as "comnet" or "community network" maintained by the IOTA community.
+
+Note that when no IOTA network is specified, it is assumed that the DID is located on the `main` network. This means that the following DIDs will resolve to the same DID Document:
+```
+did:iota:main:H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV
+did:iota:H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV
+```
+
+### Network-Shard
+
+In a future update to the network, IOTA will likely have shards containing subsets of the Tangle. While this is currently not the case, the space is in the DID is reserved and implementations should be able to handle this extra field. The value can currently be ignored. 
+
+### IOTA-Tag
+
+The IOTA tag references an indexation which resolves to the initial DID Messages. 
+
+#### Generation
+
+The following steps MUST be taken to generate a valid Tag:
+* Generate an asymmetric keypair using a supported verification method type.
+* Hash the public key using `BLAKE2b-256` and encode using base58.
+
+Note that this public key MUST also be embedded into the DID Document (See [CRUD: Create](#Create)).
+
+## DID Messages
+
+DID Documents associated to the `did:iota` method consist of a chain of data messages, also known as zero-value transactions, published to a Tangle called "DID messages". The Tangle has no understanding of "DID messages" and acts purely as an immutable database. The chain of DID messages and the resulting DID Documents must therefore be validated on the client side. 
+
+A DID Message can be part of one of two different message chains, the "Authentication Chain" (Auth Chain) and the "Differentiation Chain" (Diff Chain). The Auth Chain is a chain of "DID Auth Messages" that contain full DID Documents. The Diff Chain is a chain of "DID Diff Messages" that contain JSON objects which only list the differences between one DID Document and the next state. 
+
+### Autonomy of Auth DID Messages
+
+An Auth DID Message MUST contain a valid DID Document according to the W3C DID standard. In addition, the message has additional restrictions:
+
+* The DID Document MUST always contain atleast one or more authentication keys, in the `authentication` object of the DID Document. 
+* The first DID Document in the chain MUST contain an authentication key that, when hashed using the `Blake2b-256` hashing function, equals the tag section of the DID.
+* Auth DID Messages have at least the following attributes:
+    * prevMsg (required): The MessageId of an IOTA Message. This MessageId should reference the previous Auth DID Message that is updated by this new Auth DID Message. If this is the first message, the field should be submitted with an empty string. 
+    * diffChain (required): A valid index of the Tangle (previously known as an Address or Tag). The value points to an indexation of the Tangle, which will be the location of the Diff Chain, originating from the DID Document contained in the same DID Auth Message.
```

Comment:
How about only a boolean value and the index is the latest authentication key, hashed twice?